### ISSUE TITLE
Hover popup shows type information with invalid syntax

### ISSUE THREAD

**Issue by rchl** [microsoft/pyright#1030]
**Describe the bug**
The hover popup shows type information enclosed in a python code block. The code included in that code block does not have a valid python syntax though so some syntax highlighters will highlight it as invalid.

**To Reproduce**
Hover any Python class

**Expected behavior**
The hover popup to not have invalid syntax

**Screenshots or Code**
This is hover information for a class:
![Screenshot 2020-09-12 at 16 48 01](https://user-images.githubusercontent.com/153197/92998099-38409c00-f518-11ea-95fa-8e0b0790d498.png)
This is for a variable:
![Screenshot 2020-09-12 at 16 49 23](https://user-images.githubusercontent.com/153197/92998109-3d9de680-f518-11ea-885b-77b4ed64e1dd.png)

Note the red highlight which indicates that `(class)` is not valid according to python syntax.

The payload for similar request is:

```json
{
  "contents": {
    "kind": "markdown",
    "value": "```python\n(class) NpmClientHandler\n```\n"
  },
  "range": {
    "end": {
      "character": 22,
      "line": 56
    },
    "start": {
      "character": 6,
      "line": 56
    }
  }
}
```

**Additional context**
I'm running pyright in Sublime Text, not VSCode so syntax highlighting works a bit differently there but that doesn't change the fact that the code block does not have valid syntax.

Not sure how the type info could be presented otherwise without using invalid syntax. Something like this maybe:

```
```python\n'(class)' NpmClientHandler.foo\n```
```

**Comment by erictraut** [comment]
The current presentation of the hover text has been refined for readability, brevity, and consistency with other popular LSPs in the VS Code ecosystem. The text is meant to be "Python-like", but it does not necessarily conform to Python syntax exactly. The text is displayed in a manner that works well with VS Code, which is what it was designed for. 

If you want to suggest further refinements to the current hover text presentation, I recommend that you file an issue in the "pylance-release" repo and try to get other users of pylance to upvote your suggestion. We have been using that process to refine the usability of pylance/pyright hover text, diagnostic messages, etc.

**Comment by rchl** [comment]
You might not agree but I think this is more of a bug than an enhancement.

VSCode's highlighting is more permissive in those cases so it doesn't mark anything as invalid but it is still not highlighting correctly. Notice for example here the difference between highlighting in the source file itself and in the popup. It's completely different due to that `(function)` prefix basically breaking the highlighting:
![Screenshot 2020-09-12 at 18 10 30](https://user-images.githubusercontent.com/153197/92999689-447e2680-f523-11ea-9089-4bb97cdee269.png)

**Comment by rchl** [comment]
Sorry, that comparison wasn't fair as there is a colon after function name in the popup. But notice that if I put the popup's code in the source file (but without the `(function)` part, it's highlighted differently:
![Screenshot 2020-09-12 at 18 16 30](https://user-images.githubusercontent.com/153197/92999793-16e5ad00-f524-11ea-9c72-833a4321c67e.png)

**Comment by erictraut** [comment]
If you file an issue in the "pylance-release" repo and can get enough other users to upvote it, that's the best way to make your case.
